deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-daemonset.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: elastic-agent
+  namespace: kube-system
+  labels:
+    app: elastic-agent
+spec:
+  selector:
+    matchLabels:
+      app: elastic-agent
+  template:
+    metadata:
+      labels:
+        app: elastic-agent
+    spec:
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          effect: NoSchedule
+      serviceAccountName: elastic-agent
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      containers:
+        - name: elastic-agent
+          image: docker.elastic.co/beats/elastic-agent:%VERSION%
+          args: [
+            "-c", "/etc/agent.yml",
+            "-e", "-d", "composable.providers.kubernetes",
```

Comment:
This one has the debug selector specific to the kubernetes provider. That might be acceptable to always have on. It should be consistent across the files.

Reply:
👍🏼